marketplace/release-notes/9.0.0.txt

```diff
@@ -0,0 +1 @@
+Remove encrypt / decrypt functionality that was deprecated in 8.7.0.
```

Comment:
It might be worth mentioning which specific actions were removed, as well as what the alternative would be.

Reply:
Also, should the documentation be updated in the [**mendix/docs**](https://github.com/mendix/docs/blob/development/content/appstore/modules/community-commons-function-library.md) repository?

Reply:
I saw that the docs now mention the specific actions, so I will be resolving this thread. The part about the **mendix/docs** repo would then be out of the scope, but still something to keep in mind.